grievanceManagementSystemAPI/src/main/java/nucleusteq/com/grievance/dto/AllTicketsDto.java

```diff
@@ -0,0 +1,368 @@
+   *
+   * @return The ticketStatus.
+   */
+  public TicketStatus getTicketStatus() {
+    return ticketStatus;
+  }
+
+  /**
+   * Set the status of the ticket.
+   *
+   * @param ticketStatusParam The ticketStatus to set.
+   */
+  public void setTicketStatus(final TicketStatus ticketStatusParam) {
+    this.ticketStatus = ticketStatusParam;
+  }
+
+  /**
+   * Get the user identifier associated with the ticket.
+   *
+   * @return The userId.
+   */
+  public Integer getUserId() {
+    return userId;
+  }
+
+  /**
+   * Set the user identifier associated with the ticket.
+   *
+   * @param userIdParam The userId to set.
+   */
+  public void setUserId(final Integer userIdParam) {
+    this.userId = userIdParam;
+  }
+
+  /**
+   * Get fulname.
+   *
+   * @return the fullName
+   */
+  public String getFullName() {
+    return fullName;
+  }
+
+  /**
+   * Set fulname.
+   *
+   * @param fullNameParam the fullName to set
+   */
+  public void setFullName(
+      final String fullNameParam) {
+    this.fullName = fullNameParam;
+  }
+
+  /**
+   * Get last updated date.
+   *
+   * @return Date
+   */
+  public String getDate() {
+    return date;
+  }
+
+  /**
+   * Set last updated date.
+   *
+   * @param dateParam The date to set.
+   */
+  public void setDate(
+      final String dateParam) {
+    this.date = dateParam;
+  }
+
+  /**
+   * Get last updated time.
+   *
+   * @return The time
+   */
+  public String getTime() {
+    return time;
+  }
+
+  /**
+   * Set last updated time.
+   *
+   * @param timeParam the time to set
+   */
+  public void setTime(
+      final String timeParam) {
+    this.time = timeParam;
+  }
+
+  /**
+   * Get creation time.
+   *
+   * @return The createrTime
+   */
+  public String getCreationTime() {
+    return creationTime;
+  }
+
+  /**
+   * Set creation time.
+   *
+   * @param timeParam the creationTime to set
+   */
+  public void setCreationTime(
+      final String timeParam) {
+    this.creationTime = timeParam;
+  }
+
+  /**
+   * Get creation date.
+   *
+   * @return the creationDate
+   */
+  public String getCreationDate() {
+    return creationDate;
+  }
+
+  /**
+   * Set creation date.
+   *
+   * @param creationDateParam the creationDate to set
+   */
+  public void setCreationDate(
+      final String creationDateParam) {
+    this.creationDate = creationDateParam;
+  }
+
+  /**
+  * @return the sNo
+  */
+  public Integer getSerialNumber() {
+   return serialNumber;
+  }
+
+  /**
+  * @param sNoParam the sNo to set.
+  */
+  public void setSerialNumber(final Integer sNoParam) {
+  this.serialNumber = sNoParam;
+  }
+
+   /**
+   * Ticket details.
+   */
+  @Override
+  public String toString() {
+    return "TicketDto [ticketId=" + ticketId + ", title="
+        + title + ", description=" + description + ", department="
+        + department + ", ticketType=" + ticketType + ", ticketStatus="
+        + ticketStatus + ", userId=" + userId
+        + "]";
+  }
+}
```

Comment:
Check if you are required to add hash and equals methods to dtos and entities. Add them if it is.